weblate/trans/views/js.py

```diff
@@ -151,7 +151,13 @@ def ignore_check_source(request, check_id):
         unit.save(same_content=True)
 
     # response for AJAX
-    return HttpResponse("ok")
+    return JsonResponse(
+        {
+            "extra_flags": unit.extra_flags,
+            "all_flags": unit.all_flags.format(),
```

Comment:
Need to parse the translation unit flags here, not the source one (it might have extra ones, for example source string might be read-only):

```suggestion
            "all_flags": obj.unit.all_flags.format(),
```